newIDE/app/src/Profile/Achievement/AchievementList.js

```diff
@@ -0,0 +1,90 @@
+// @flow
+import React, { useEffect, useState } from 'react';
+import { DateFormat } from '@lingui/macro';
+
+import { Column, Line } from '../../UI/Grid';
+import Text from '../../UI/Text';
+import {
+  type Badge,
+  type Achievement,
+} from '../../Utils/GDevelopServices/User';
+
+type Props = {|
+  badges: Array<Badge>,
+  achievements: Array<Achievement>,
+|};
+
+const achievementBaseline = {
+  width: '100%',
+  padding: '0 20px',
+};
+
+const styles = {
+  achievementsContainer: {
+    maxHeight: 250,
+    overflowY: 'scroll',
+  },
+  unlockedAchievement: {
+    ...achievementBaseline,
+    color: 'green',
+    display: 'flex',
+    justifyContent: 'space-between',
+  },
+  lockedAchievement: {
+    ...achievementBaseline,
+    color: '#CCCCCC',
```

Comment:
For the color:
- For something locked, prefer reducing the opacity so that the theme color is preserved.
- For unlocked, I would use something from the theme. For example, `message.valid`.

```
const gdevelopTheme = React.useContext(ThemeContext);
//...
  style={{...styles.something, color: gdevelopTheme.messages.valid}}
```